### ISSUE TITLE
[Bug Report] RaidManager error

### ISSUE THREAD

**Issue by zivilyyyn** [Merathilis/ElvUI_MerathilisUI#18]
### What is the issue you are having?

Get error when I enter the game. Cannot find MerathilisUI options in ElvUI option. And get error message
![WoWScrnShot_050122_224851](https://user-images.githubusercontent.com/32798348/166151354-f9e5d220-fc7c-4dea-8152-2a44d1d3cbda.jpeg)
.

### What is the expected behavior?

fix this error.

### What actually happened?

Cannot find MerathilisUI options in ElvUI option.

### Errors

```shell
Message: ...hilisUI\Mainline\Modules\RaidManager\RaidManager.lua:264: attempt to index local 'WorldMarkButton' (a nil value)
Time: Sun May  1 22:44:52 2022
Count: 1
Stack: ...hilisUI\Mainline\Modules\RaidManager\RaidManager.lua:264: attempt to index local 'WorldMarkButton' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\ElvUI_MerathilisUI\Mainline\Modules\RaidManager\RaidManager.lua"]:264: in function `CreateRaidManager'
[string "@Interface\AddOns\ElvUI_MerathilisUI\Mainline\Modules\RaidManager\RaidManager.lua"]:684: in function `Initialize'
[string "@Interface\AddOns\ElvUI_MerathilisUI\Core\Init.lua"]:102: in function `InitializeModules'
[string "@Interface\AddOns\ElvUI_MerathilisUI\Core\Init.lua"]:126: in function `?'
[string "@Interface\AddOns\ElvUI\Libraries\Core\LibElvUIPlugin-1.0\LibElvUIPlugin-1.0.lua"]:294: in function <...aries\Core\LibElvUIPlugin-1.0\LibElvUIPlugin-1.0.lua:290>
[string "=[C]"]: in function `Initialize'
[string "@Interface\AddOns\ElvUI\Core\init.lua"]:218: in function <Interface\AddOns\ElvUI\Core\init.lua:217>
[string "=[C]"]: ?
[string "@Interface\AddOns\ElvUI\Libraries\Core\Ace\AceAddon-3.0\AceAddon-3.0.lua"]:70: in function <...vUI\Libraries\Core\Ace\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Interface\AddOns\ElvUI\Libraries\Core\Ace\AceAddon-3.0\AceAddon-3.0.lua"]:527: in function `EnableAddon'
[string "@Interface\AddOns\ElvUI\Libraries\Core\Ace\AceAddon-3.0\AceAddon-3.0.lua"]:630: in function <...vUI\Libraries\Core\Ace\AceAddon-3.0\AceAddon-3.0.lua:615>

Locals:
```

### ElvUI Changelog/Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://github.com/Merathilis/ElvUI_MerathilisUI/blob/development/CHANGELOG.md) and done the [Troubleshooting Steps](https://github.com/Merathilis/ElvUI_MerathilisUI/wiki/Troubleshooting).

**Comment by Merathilis** [comment]
The error you are getting makes no sense. Cause the Raid Marker is not loaded if you are in the open World and not in a group/raid.

You also have other AddOns enabled in that Screenshot, please to the troubleshooting steps first before you open a Ticket.

**Comment by Merathilis** [comment]
If you still have that issue after you done the troubleshooting steps, please download my dev. version and test with this. I just protected the Button that error a bit: https://github.com/Merathilis/ElvUI_MerathilisUI/archive/refs/heads/development.zip

**Comment by zivilyyyn** [comment]
The error didn't show when I use the dev. version. And I got a system message said that "It seems that the Blizzard_CompactRaidFrame is disabled by one of your addon and this will cause error and problem. it will be re-enabled now. Then I click ok and the UI reloaded. The error didn't show. 

And then I switched back to the stable version, the error still didn't show up. I think maybe it's some other addons I used cause this error. 

But the error still exist when I log in other characters. It seems that I show re-enable it for all of my characters.

Thank you for your help.  :)
